app/views/content/is-teaching-right-for-me/engineers-teach-physics/_article.html.erb

```diff
@@ -123,8 +123,7 @@
   </section>
 
   <section class="col col-720">
-    <h2 class="heading-s">Funding for teaching physics</h2>
-
+<h2>Fund your teaching training</h2>
```

Comment:
@gemmadallmandfe @caitlin-reidy I wondered if it might make it more consistent if we say Next steps into teacher training so it's the same as the other page physics page (and if we should also have this on the maths page?) and then have the Fund your teacher training H2?